index.html

```diff
@@ -281,7 +283,46 @@ <h2 class="lets-connect">LET’S CONNECT</h2>
           </ul>
         </div>
       </section>
-      <section class="contact-me"></section>
+      <section class="contact-me">
+        <h1 class="contact-me-header">Contact me</h1>
+        <p class="contact-me-description-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form
+          action="https://formspree.io/f/xpznodbr"
+          method="post"
+          class="contact-form"
+        >
+          <input
+            type="text"
+            name="name"
+            class="form-input"
+            placeholder="Yeremia"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="email"
+            class="form-input"
+            placeholder="natajayanj14@gmail.com"
+            required
+          />
```

Comment:
- [ ] **[OPTIONAL]:** I would suggest changing these placeholders with something more generic that describes the actual field 📜 . For example: `"Enter your name..."` and `"Enter your Email..."`. This will provide better guidance for users and also assistive technologies such as screen readers. 🤓 🚀